modules/ocf_irc/files/inspircd.service.d/override.conf

```diff
@@ -0,0 +1,7 @@
+[Unit]
```

Comment:
Nit: consider naming this file more descriptively, e.g. `inspircd-group-restart.conf`